cryptoportfoliotracker.py

```diff
@@ -89,16 +86,16 @@ def main():
     """
     coins = None
     try:
-        cli._cls()
+        cli._cls()  # type: ignore
```

Comment:
`#type: ignore` is great for getting Pylance to shut up about stubs or untyped 3rd party libs, the goal is to type your own stuff, not try to please the linter for 100% coverage of types for the sake of it.